pkg/tfbridge/x/apply.go

```diff
@@ -25,6 +25,9 @@ import (
 )
 
 // Add mapped resources and datasources according to the given strategies.
+//
+// Deprecated: This item has been moved to
+// "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/defaults.ApplyStrategy"
 func ComputeDefaults(info *b.ProviderInfo, opts DefaultStrategy) error {
```

Comment:
Can it call the new implementation instead of leaving the duplicated code body here?

Reply:
No, since it would be a cyclical dependency between `tfbridge/x` and `tfbridge`.